### PR TITLE
Clean up TestLoopFuser.cxx

### PR DESCRIPTION
**Pull request by adayton1**
* TestLoopFuser.cxx had EXPECT_EQ in a loop, which would spew out tons of output
* I changed that to ASSERT_EQ, which works in vanilla for loops, but not RAJA loops
* General clean up

### CODE REVIEW

**Review by robinson96 — APPROVED** [review]
LGTM

File: test/TestLoopFuser.cxx [hunk 1]
```diff
@@ -78,12 +78,12 @@ GPU_TEST(TestPacker, packFixedRange) {
...
    // host data should not be updated yet
-   int * host_dst = dst.getPointer(care::CPU, false);
-   int * host_src = src.getPointer(care::CPU, false);
+   const int* host_src = src.data(chai::CPU, false);
+   const int* host_dst = dst.data(chai::CPU, false);
 
-   for (int i = 0; i < 2; ++i) {
-      EXPECT_EQ(host_dst[i], -1);
-      EXPECT_EQ(host_src[i], i);
+   for (int i = 0; i < arrSize; ++i) {
```
Comment (liu15) [inline test/TestLoopFuser.cxx:84]: this was only checking a small bit before?

**Review by liu15 — APPROVED** [review]

File: test/TestLoopFuser.cxx (hunk 1 above)
Comment (adayton1) [reply test/TestLoopFuser.cxx:84]: Yeah, I guess so.

Comment (liu15) [reply test/TestLoopFuser.cxx:84]: Probably because of the huge spam on failure :)

Comment (dtaller) [reply test/TestLoopFuser.cxx:84]: Good catch!

File: test/TestLoopFuser.cxx [hunk 2]
```diff
@@ -142,189 +146,239 @@ GPU_TEST(TestPacker, packFixedRangeMacro) {
 
    care::gpuDeviceSynchronize(__FILE__, __LINE__);
 
+   const int* host_src;
+   const int * host_dst;
```
[the remark this answers is not in the document]
Comment (adayton1) [reply test/TestLoopFuser.cxx]: Fixed.

**Review by dtaller — APPROVED** [review]
Looks good to me, aside from the tiny formatting thing.

**Pull request merged**